spk/ps3netsrv/Makefile

```
@@ -1,16 +1,15 @@
 SPK_NAME = ps3netsrv
-SPK_VERS = 1.47.46
-SPK_REV = 9
+SPK_VERS = 20240709
+SPK_REV = 10
 SPK_ICON = src/${SPK_NAME}.png
 
 DEPENDS = cross/$(SPK_NAME)
```

Just a small change in preparation of #6255
The dependency list creation will not allow make variables in DEPENDS definitions

```suggestion
DEPENDS = cross/ps3netsrv
```